bool call
int8 weight
string target
int8 startTime
int8 endTime